nvflare/private/fed/app/fl_conf.py

```diff
@@ -35,7 +35,7 @@
 from .fl_app_validator import FLAppValidator
 
 FL_PACKAGES = ["nvflare"]
-FL_MODULES = ["server", "client", "app_common", "private", "app_opt"]
```

Comment:
we need to check all our examples / tutorials / notebooks to make sure we are not using "name" to refer to components in "app_opt"

@SYangster @nvkevlu @holgerroth